Random improvements

oguzkocer opened the pull request:
This is a pretty random PR. I have seen a bunch of issues while looking for something across the project. I thought I'd improve some of them and send a PR. I am not really sure if it makes sense or not. I realized that I could keep going, but even if we wanted to do a big cleanup it's better to do it in smaller chunks to avoid a big PR review even if it's small changes. So, let's see how this one goes.

Things changed in this PR:
1. Remove some unused methods. (I really like to keep the code clean as much as possible and I personally prefer removing unused methods even if we _might_ need it in the future as it gets much harder to navigate the project with all the unused random stuff)
2. I think we tend to use `public` methods too much even if we don't use it anywhere else but that same file. I like to use the most-restrictive level as much as possible, so there are some changes around that.
3. 120 char limit :)
4. Spelling fixes
5. Removing unused parameters
6. Changing an iterator to for-each

Please let me know what you think about this kind of PR. This is an experiment for me and if we don't want to merge PRs like this it's completely fine, we can just close it. Thanks!

tonyr59h commented:
Always a pleasure to see ![](https://cldup.com/itWN00t5wY.png)

tonyr59h commented on WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java:
```diff
@@ -114,25 +86,14 @@ private static Intent prepareGalleryIntent(String title) {
-    }
-
-    private static Intent preparelaunchCamera(Context context, String applicationId, LaunchCameraCallback callback) {
+    private static Intent prepareLaunchCamera(Context context, String applicationId, LaunchCameraCallback callback) {
```
Wow, surprised you even noticed this one!

tonyr59h commented on libs/utils/WordPressUtils/src/main/java/org/wordpress/android/util/GravatarUtils.java:
```diff
@@ -65,6 +65,7 @@ public static String fixGravatarUrl(final String imageUrl, int avatarSz, Default
     public static String gravatarFromEmail(final String email, int size) {
         return gravatarFromEmail(email, size, DEFAULT_GRAVATAR);
     }
+
```
👍

tonyr59h commented:
Good changes all-around. I agree that it's important to keep the code clean. There's a ton of maintenance to do yet, thanks for keeping it small/light 😄 

This one is good to :shipit: once the merge conflicts have been dealt with.

oguzkocer commented:
@tonyr59h Thanks for the review! I've merged `develop` in and fixed the conflict. I have also noticed that there was an extra import in `EditPostSettingsFragment` which was a result of recent changes, so I've removed that as well.

tonyr59h commented:
:shipit:
